frontend/src/themes/alby.css

```diff
@@ -1,5 +1,6 @@
 .theme-alby {
   --background: 0 0% 100%;
+  --intro-background: 240 10% 3.9%;
```

Comment:
why do we need a new variable?

and best to wait for @stackingsaunter to confirm what we should do